extensions/amp-story-interactive/0.1/amp-story-interactive-slider.css

```diff
@@ -95,6 +99,115 @@
   border-bottom-right-radius: 50% !important;
 }
 
+.i-amphtml-story-interactive-slider-average-indicator {
+  transform: translate(-50%, -50%) scale(0) !important;
+  width: 2em !important;
+  height: 2em !important;
+  top: 50% !important;
+  left: var(--average) !important;
+  opacity: 0.6 !important;
+  border-radius: 50%;
+  position: absolute;
+}
+
+.i-amphtml-story-interactive-post-selection .i-amphtml-story-interactive-slider-average-indicator::after,
+.i-amphtml-story-interactive-post-selection .i-amphtml-story-interactive-slider-average-indicator::before {
+  --animation-duration: 3s;
+  transform: scale(0);
+  box-sizing: border-box;
+  content: '';
+  width: 2em;
+  height: 2em;
+  position: absolute;
+  box-shadow: inset 0 0 0 0.05em var(--i-amphtml-interactive-strong-text-color);
+  border-radius: 50%;
+  animation: pulse var(--animation-duration) linear infinite forwards !important;
+}
+
+.i-amphtml-story-interactive-post-selection .i-amphtml-story-interactive-slider-average-indicator::before {
+  animation-delay: calc(var(--animation-duration) / 2) !important;
+}
+
+@keyframes pulse {
+  0% {
+    transform: scale(0.2);
+    opacity: 0;
+    box-shadow: inset 0 0 0 0.6em var(--i-amphtml-interactive-strong-text-color);
+  }
+  10% {
+    opacity: 1;
+  }
+  20% {
+    opacity: 1;
+    box-shadow: inset 0 0 0 0.3em var(--i-amphtml-interactive-strong-text-color);
+  }
+  60% {
+    opacity: 1;
+  }
+  100% {
+    transform: scale(1);
+    opacity: 0;
+    box-shadow: inset 0 0 0 0.01em
+      var(--i-amphtml-interactive-strong-text-color);
+  }
+}
+
+.i-amphtml-story-interactive-post-selection .i-amphtml-story-interactive-slider-average-indicator {
+  transform: translate(-50%, -50%) !important;
+}
```

Comment:
Please move this to be under the block for`.i-amphtml-story-interactive-slider-average-indicator` (line 113).
That will make the file easier to read since this is a modifier.

Please also add `scale(1)` to this since that's the thing that's changing, so it will then be:
`transform: translate(-50%, -50%) scale(1) !important;`
This is only working now because resetting scale is a side-effect of updating transform.